jujugui/static/gui/src/app/app.js

```diff
@@ -639,6 +639,47 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Sends the discharge token via POST to the storefront. This is used
+      when the GUI is operating in GISF mode, allowing a shared login between
+      the GUI and the storefront.
+
+      @method _sendGISFPostBack
```

Comment:
Just an FYI - with the latest jsdoc we don't need to define the function name (not that we use any doc gen tool right now) 😄